### PR TITLE
Dev screpyy96 connect section completed

### PR DESCRIPTION
**Pull request by screpyy96**

### CODE REVIEW

File: src/_component/_connect/connect.pug [hunk 1]
```diff
@@ -0,0 +1,48 @@
+-var sectionTitle= data.connect.sectionTitle
+include ../section-title/section-title.pug
+
+.column(span="row")
+    .connect__avatar
+.column(span="row" height="10")
+
```
Comment (RalucaRbc) [inline src/_component/_connect/connect.pug:7]: You can delete one of these 2 empty lines :)

File: src/_component/_connect/connect.scss [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+.about-box__footer {
+  display: flex;
+}
+
+.about-box__body {
+  border-top: 2px solid #f5f5f5;
+  border-bottom: 2px solid #f5f5f5;
+}
```
Comment (RalucaRbc) [inline src/_component/_connect/connect.scss:103]: You need an empty line here 😃

File: src/_component/_connect/connect.scss [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+@for $i from 1 through 10 {
+  .connect__radio:nth-of-type(#{$i}):checked
+    ~ [data-about-box="#{$i}"] {
+    display: block;
+  }
+}
+
+[data-about-box] {
+  display: none;
+}
```
Comment (RalucaRbc) [inline src/_component/_connect/connect.scss:122]: Empty line at the end of the file for readability

File: src/styles.scss [hunk 4]
```diff
@@ -2,3 +2,5 @@
 @import "_component/_layout/roboto";
 @import "./_component//about-me/about-me.scss";
 @import "./_asset/font/style.scss";
+@import "./_component/_connect/connect.scss";
```
Comment (RalucaRbc) [inline src/styles.scss:5]: Be careful with these imports and try to use the generic ones at the top and the most specific ones at the bottom; like the one in which you import the font first and then the one responsible the about-me section

File: src/styles.scss [hunk 5]
```diff
@@ -2,3 +2,5 @@
 @import "_component/_layout/roboto";
 @import "./_component//about-me/about-me.scss";
```
Comment (RalucaRbc) [inline src/styles.scss:3]: Think you have an extra `/` between `_component` and `about-me`

**Review by RalucaRbc — APPROVED** [review]

**Review by MariaTeo — APPROVED** [review]
💯

**Pull request merged**